package.json

```diff
@@ -7,9 +7,10 @@
   },
```

Comment:
The npm version probably needs to change to please package manager and deployment systems.